### PR TITLE
Project- Mobile menu

### PR DESCRIPTION
**Pull request by sja-thedude**
- Javascript Linter added
- Mobile menu created using Javascript
- CSS File updated for the mobile menu
- Html File was updated with the links for displaying the correct part of the project

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -159,6 +163,47 @@ h1:hover {
...
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  color: #edecff;
+  background-image: url("img/shapes-mobile-menu.png");
+  background-repeat: no-repeat;
+  background-size: 992px;
+  margin: 0;
+  padding-top: 12%;
```
Comment (elisha2kyakpo1) [inline style.css:193]: - [ ] The project works very well, but I saw some differences in styling, the first one being the page overflow. See below
overflow

- [ ] If you look properly, you will see that the image below has white images on it. you should follow the given design.
popup

- [ ] The white images look different from the original, you can see from the images below.

Your design
white-img1

Original
white-img2

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi, @sja-thedude,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
You have tried to implement almost all the requirements, keep it up.


## Required Changes ♻️
_Check the comments under the review._👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (rloterh) [reply style.css]: The mentioned aspects need further improvement. Kindly ensure to improve this:
- [ ] The hamburger menu is not is visible on mobile screen mode, and there some page overflow that still exits

**Your current mobile screen (Browser mobile mode):**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi SJA, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments above, made under the reviews of the previous code reviewer(s). Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (sja-thedude) [reply style.css]: 3F48D553-A678-49E9-BC93-9C0D989F51C2

But it is working perfectly here!!!

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @sja-thedude, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**